pipeline/inferrer/palette_inferrer/app/palette_encoder.py

```diff
@@ -1,202 +1,87 @@
-from collections import Counter
 import numpy as np
-from sklearn.cluster import KMeans
-from joblib import Parallel, delayed
+from PIL import Image
 
-from .color import rgb_to_hsv, rgb_to_hex
 
-tau = 2 * np.pi
-min_bin_width = 1.0 / 256
+class PaletteEncoder:
```

Comment:
Think this will need a `__call__` defined so it plugs into the batch thing - very straightforward

Reply:
Already done!